### PR TITLE
[wip] Coordinate transform example

### PR DESCRIPTION
**Pull request by vpuri3**
To do:
1. fix latex rendering
2. add image

### CODE REVIEW

File: docs/src/pinn/poisson_annulus.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+## Coordinate Transform and Differential Operators
+
+We represent the field vairable, ``u``, and physical coordinates ``x,y`` in terms of reference variables ``r,\theta``:
+
+```math
+u(r,\theta), \, x(r,\theta), \, y(r,\theta)
+```
+
+```julia
+@paameters r,\theta
```
Comment (KirillZubov) [inline docs/src/pinn/poisson_annulus.md:158]: @parameters

Comment (vpuri3) [reply docs/src/pinn/poisson_annulus.md]: fixed :D

**Comment by KirillZubov** [comment]
very cool example

**Comment by vpuri3** [comment]
Thanks. Could you help me with the writeup? LaTeX isn't compiling in the markdown, and I don't know where to upload an image of the solution.

**Comment by KirillZubov** [comment]
> Thanks. Could you help me with the writeup? LaTeX isn't compiling in the markdown, and I don't know where to upload an image of the solution.
documenter.jl compiles in LaTeX after merge. so it's fine. the markdown don't do it.

**Comment by KirillZubov** [comment]
the same in `main` branch https://github.com/SciML/NeuralPDE.jl/blob/master/docs/src/pinn/2D.md but in docs it is already compiled

**Comment by vpuri3** [comment]
Thanks. Where can I upload the image?
poisson_annulus

**Comment by vpuri3** [comment]
Nevermind. Dragging and dropping into the markdown worked!

**Comment by vpuri3** [comment]
@ChrisRackauckas , I think this commit is ready to merge.

**Comment by ChrisRackauckas** [comment]
Yeah, just put images into the github and then use `` to put it into the docs.

File: docs/src/pinn/poisson_annulus.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
+# Poisson Equation
+
+In this example, we solve the Poisson equation on an annulus to demostrate how NeuralPDE can solve differential equations in complex geometries.
```
Comment (ChrisRackauckas) [inline docs/src/pinn/poisson_annulus.md:3]: using coordinate maps, etc. This should get a bit more explanation at the beginning, with a nice source that goes into more depth, etc.

**Comment by ChrisRackauckas** [comment]
We should really find a good way to do such a coordinate transformation automatically symbolically.

**Pull request closed without merging**